src/shared/RuntimeInfoManager.ts

```diff
@@ -0,0 +1,81 @@
+import path from "path";
+import { EngineId, EngineInfo } from "@/type/preload";
+/**
+ * サードパーティ向けランタイム情報のエクスポート情報クラス
+ */
+export class RuntimeInfoManager {
+  private runtimeInfoPath: string;
+
+  constructor(userDataPath: string) {
+    this.runtimeInfoPath = path.join(userDataPath, "runtime-info.json");
+  }
+
+  /**
+   * サードパーティ向けランタイム情報のファイルパスを取得する
+   */
+  public getRuntimeInfoPath(): string {
+    return this.runtimeInfoPath;
+  }
+}
+
+/**
+ * サードパーティ向けランタイム情報のレコード定義
+ * Note:変更時はOutputInfoDataFor3rdPartyのバージョン定義も変更すること
+ */
+export interface EngineInfoRecordFor3rdParty {
+  uuid: EngineId;
+  url: string;
+  name: string;
+}
+
+/**
+ * サードパーティ向けランタイム情報のレコード定義
+ * Note:変更時はOutputInfoDataFor3rdPartyのバージョン定義も変更すること
+ */
+export interface EngineInfoFormatFor3rdParty {
```

Comment:
ここはエンジン情報というよりそれを含んだ情報全部なので、こうかもです！
```suggestion
export interface RuntimeInfoFormatFor3rdParty {
```